packages/next-app/src/app/internalmanager/(components)/TokenSelect.tsx

```diff
@@ -212,7 +314,9 @@ function TokenRow({
     <Table.BodyRow key={token.tokenAddress}>
       <Table.BodyCell customWidth="w-12">
         <Link
-          href={`/internalmanager/${chainName}/${operationKind}/${token.tokenAddress}`}
+          href={
+            `/internalmanager/${chainName}/${operationKind}/${token.tokenAddress}` as Route
```

Comment:
there's something off if this isn't been recognized as a route -- lowpriority #techdebt ticket imo